kbds/keyboard-configuration.yaml

```diff
@@ -1,101 +1,248 @@
 # File generated by ./scripts/keyboard-scraper.sh, please do not edit manually
```

Comment:
Should we update/drop this line?

Reply:
Yes, it's slightly incorrect now with
* more input data
* passing through pyyaml for normalization and merging

I will leave for now, as we are considering generating at build time.  It's still mostly correct.

Reply:
Before I forget, this is how I constructed the full list of locales from the .arb files at https://github.com/canonical/ubuntu-flutter-plugins/tree/main/packages/ubuntu_localizations/lib/src/l10n

```bash
for arb in *.arb; do
    lang=$(jq -r '.["@@locale"]' $arb | cut -d_ -f1)
    country=$(jq -r '.["countryCode"]' $arb)
    echo "${lang}_${country}.UTF-8"
done
```